keepassxc tests to check if it is building from a git repo and assumes
it is a developer build. When building in jenkins, it does not think
it is a release build.  Remove this check.

--- keepassxc-2.7.10/CMakeLists.txt.old	2025-04-19 12:42:58.472981419 -0400
+++ keepassxc-2.7.10/CMakeLists.txt	2025-04-19 12:43:43.535473174 -0400
@@ -128,33 +128,6 @@
 set(KEEPASSXC_BUILD_TYPE "Snapshot" CACHE STRING "Set KeePassXC build type to distinguish between stable releases and snapshots")
 set_property(CACHE KEEPASSXC_BUILD_TYPE PROPERTY STRINGS Snapshot Release PreRelease)
 
-# Retrieve git HEAD revision hash
-set(GIT_HEAD_OVERRIDE "" CACHE STRING "Manually set the Git HEAD hash when missing (eg, when no .git folder exists)")
-execute_process(COMMAND git rev-parse --short=7 HEAD
-        WORKING_DIRECTORY ${CMAKE_SOURCE_DIR}
-        OUTPUT_VARIABLE GIT_HEAD
-        ERROR_QUIET)
-string(STRIP "${GIT_HEAD}" GIT_HEAD)
-if(GIT_HEAD STREQUAL "" AND NOT GIT_HEAD_OVERRIDE STREQUAL "")
-    string(SUBSTRING "${GIT_HEAD_OVERRIDE}" 0 7 GIT_HEAD)
-elseif(EXISTS ${CMAKE_SOURCE_DIR}/.gitrev)
-    file(READ ${CMAKE_SOURCE_DIR}/.gitrev GIT_HEAD)
-endif()
-message(STATUS "Found Git HEAD Revision: ${GIT_HEAD}\n")
-
-# Check if on a tag, if so build as a release
-execute_process(COMMAND git tag --points-at HEAD
-        WORKING_DIRECTORY ${CMAKE_SOURCE_DIR}
-        OUTPUT_VARIABLE GIT_TAG
-        ERROR_QUIET)
-string(REGEX REPLACE "latest" "" GIT_TAG "${GIT_TAG}")
-if(GIT_TAG MATCHES "[0-9]+\.[0-9]+\.[0-9]+")
-    string(STRIP "${GIT_TAG}" GIT_TAG)
-    set(OVERRIDE_VERSION ${GIT_TAG})
-elseif(EXISTS ${CMAKE_SOURCE_DIR}/.version)
-    file(READ ${CMAKE_SOURCE_DIR}/.version OVERRIDE_VERSION)
-endif()
-
 string(REGEX REPLACE "(\r?\n)+" "" OVERRIDE_VERSION "${OVERRIDE_VERSION}")
 if(OVERRIDE_VERSION)
     if(OVERRIDE_VERSION MATCHES "^[\\.0-9]+-beta[0-9]*")
